SocialCareCaseViewerApi.Tests/V1/Controllers/SocialCareCaseViewerApiControllerTests.cs

```diff
@@ -68,6 +71,21 @@ public void ListContactsReturns400WhenQueryParametersAreInvalid()
             response.Value.Should().Be("Invalid Parameters");
         }
 
+        [Test]
+        public void CreateAllocationReturns201WhenSuccessful()
+        {
+            var request = _fixture.Create<CreateAllocationRequest>();
+
+            __mockCreateAllocationUseCase.Setup(x => x.Execute(request))
+                .Returns(request);
```

Comment:
This mock is returning the request object instead of what the use case should actually return

Reply:
This will be resolved when the return object is defined